src/utils.ts

```diff
@@ -4,11 +4,23 @@ import { readFile as readFileCb } from 'fs'
 import { promisify } from 'util'
 import config from 'config'
 import { hexToAscii } from 'web3-utils'
+import BigNumber from 'bignumber.js'
 
 import { Application, Config, isSupportedServices, Logger, SupportedServices } from './definitions'
 
 const readFile = promisify(readFileCb)
 
+/**
+ * Bignumber.js utils functions
+ */
+export function bn (v: any): BigNumber {
```

Comment:
I am not a big fan of this utility function but the call is on @AuHau & @jurajpiar . To me it hinders readability and you can use `new BigNumber(...)` or even just `BigNumber(...)`

Reply:
Fixed

Reply:
fixed